src/Test/PdbUtilities/Pdb/SymReaderFactory.cs

```diff
@@ -23,7 +23,7 @@ public static class SymReaderFactory
         [DllImport("Microsoft.DiaSymReader.Native.amd64.dll", EntryPoint = "CreateSymReader")]
         private extern static void CreateSymReader64(ref Guid id, [MarshalAs(UnmanagedType.IUnknown)]out object symReader);
 
-        private static ISymUnmanagedReader3 CreateNativeSymReader(Stream pdbStream, object metadataImporter)
+        private static ISymUnmanagedReader5 CreateNativeSymReader(Stream pdbStream, object metadataImporter)
```

Comment:
In the past there were cases where the compiler was installed without the specific version of native diasymreader that we needed.  That meant it could be older.  Is that the case anymore?  If so do we have to worry about cases where ISymUnmanagedReader5 is not avialable?